assets/dist/static/js/adapters.493e488e.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('eventespresso-hooks', 'react', 'react-dom', 'wp-i18n', 'wp-keycodes'), 'version' => '300e4d6f686bb325677182d796c69181');
```

Comment:
missing EOL

Reply:
this is a prod file built via barista and pushed to the core repo, so will have to be fixed on barista side

Reply:
Since those files are auto-generated, we can ignore them in PHP lint.